The git utility is required by processoer

lslebodn opened the pull request:
Traceback (most recent call last):
    File "/src/scripts/download-unpack-build.py", line 42, in <module>
      utils.download_source(build_info, output_source_dir)
    File "/src/assayist/processor/utils.py", line 169, in download_source
      _assert_command('git')
    File "/src/assayist/processor/utils.py", line 35, in _assert_command
      raise RuntimeError(f'The command "{cmd_name}" is not installed and is required')
  RuntimeError: The command "git" is not installed and is required

mprahl commented on docker/Dockerfile:
```diff
@@ -14,6 +14,7 @@ RUN dnf -y install \
   --setopt=tsflags=nodocs \
   # Install brewkoji for the koji.conf.d config file
   brewkoji \
+  git \
```
Could you use `git-core` instead? It should make the image smaller.

lslebodn replied:
Nice idea

mprahl commented:
@lslebodn could you please fix the typo in the commit and PR title (`processoer` => processor`)? Ideally, it should be `analyzer` though.

mprahl commented:
@lslebodn also, could you fix your commit's authorship to match your GitHub username and email?

lslebodn committed:
The git utility is required by some analyzers

  Traceback (most recent call last):
    File "/src/scripts/download-unpack-build.py", line 42, in <module>
      utils.download_source(build_info, output_source_dir)
    File "/src/assayist/processor/utils.py", line 169, in download_source
      _assert_command('git')
    File "/src/assayist/processor/utils.py", line 35, in _assert_command
      raise RuntimeError(f'The command "{cmd_name}" is not installed and is required')
  RuntimeError: The command "git" is not installed and is required

lslebodn commented:
> could you please fix the typo in the commit and PR title (processoer => processor)? Ideally, it should beanalyzer` though.

I used analyzer

> also, could you fix your commit's authorship to match your GitHub username and email?

done as well.

TY for review

mprahl reviewed (approved):
:+1:
